specification/botservice/resource-manager/Microsoft.BotService/stable/2021-03-01/botservice.json

```diff
@@ -0,0 +1,2640 @@
+            "$ref": "./examples/PutChannel.json"
+          },
+          "Create Alexa Bot": {
+            "$ref": "./examples/PutAlexaChannel.json"
+          },
+          "Create Line Bot": {
+            "$ref": "./examples/PutLineChannel.json"
+          },
+          "Create DirectLine Speech Bot": {
+            "$ref": "./examples/PutDirectLineSpeechChannel.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/channelNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BotChannel"
+            },
+            "description": "The parameters to provide for the created bot."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is created successfully or already existed, the service should return 200 (OK).",
+            "schema": {
+              "$ref": "#/definitions/BotChannel"
+            }
+          },
+          "201": {
+            "description": "If resource is created successfully, the service should return 201 (Created). Execution to continue asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/BotChannel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Channel"
+        ],
+        "description": "Updates a Channel registration for a Bot Service",
+        "operationId": "Channels_Update",
+        "x-ms-examples": {
+          "Update Bot": {
+            "$ref": "./examples/UpdateChannel.json"
+          },
+          "Update Alexa": {
+            "$ref": "./examples/UpdateAlexaChannel.json"
+          },
+          "Update Line": {
+            "$ref": "./examples/UpdateLineChannel.json"
+          },
+          "Update DirectLine Speech": {
+            "$ref": "./examples/UpdateDirectLineSpeechChannel.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/channelNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
```

Comment:
why are you flattening here?

Reply:
not changed from previous swagger version